lib/arc.js

```diff
@@ -57,18 +57,14 @@ export function configure(options) {
 export async function getDeployedContracts() {
   const contracts = (await getSettings()).daostackContracts;
   return {
-    allContracts : contracts,
+    allContracts: contracts,
     schemes: [
-      contracts.SchemeRegistrar
-      , contracts.UpgradeScheme
-      , contracts.GlobalConstraintRegistrar
-      , contracts.ContributionReward
-    ],
-    votingMachines: [
-      contracts.AbsoluteVote
+      contracts.SchemeRegistrar,
```

Comment:
Why are the schemes under votingMachines?

Reply:
@leviadam  I think you are seeing an illusion in the diff output.  Only AbsoluteVote is under votingMachines.